tests/test_utils.py

```diff
@@ -15,23 +15,27 @@ def test_removeall_list():
     assert removeall(4, []) == []
     assert removeall(4, [1, 2, 3, 4]) == [1, 2, 3]
     assert removeall(4, [4, 1, 4, 2, 3, 4, 4]) == [1, 2, 3]
+    assert removeall(1,[2,3,4,5,6]) == [2,3,4,5,6]
```

Comment:
After a comma, you should be adding a space. This is the case for all the additions in this PR.

Other than that, nice work!

Reply:
@MrDupin Thank You for your review

Reply:
Remember to add spaces in all the additions you made.

Reply:
@MrDupin all test cases are perfectly added now